python_modules/dagster/dagster/_core/definitions/definitions_load_context.py

```diff
@@ -64,11 +65,24 @@ def load_type(self) -> DefinitionsLoadType:
         """DefinitionsLoadType: Classifier for scenario in which Definitions are being loaded."""
         return self._load_type
 
+    @property
+    def cacheable_asset_data(self) -> Mapping[str, Sequence[AssetsDefinitionCacheableData]]:
```

Comment:
consider a name like `legacy_cacheable_asset_data` or making it private since this is user-facing

Reply:
It is currently private (no `@public`) which is how we define public methods on user-facing classes.

`_`-prefixing wouldn't make sense since we do need to access it internally outside the class def.